canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.js

```diff
@@ -90,6 +90,7 @@ export default class SVGCanvasD3 {
 				this.config.enableDragWithoutSelect !== config.enableDragWithoutSelect ||
 				this.config.enableParentClass !== config.enableParentClass ||
 				this.config.enableHightlightNodeOnNewLinkDrag !== config.enableHightlightNodeOnNewLinkDrag ||
```

Comment:
Should this be renamed as well? I see https://github.com/elyra-ai/canvas/issues/503 has been fixed and you're updating the examples in this PR

Reply:
Yes, good catch. I found some other places with that typo. None will cause any runtime error but I fixed them nonetheless.